addons/web/static/src/js/core/utils.js

```diff
@@ -500,6 +500,18 @@ var utils = {
     isComponent: function (value) {
         return value.prototype instanceof owl.Component;
     },
+    /**
+     * Checks if a keyboard event concerns
+     * the numpad decimal separator key
+     * while the NumLock modifier state is active.
+     * 
+     * @param {KeyboardEvent} ev
+     * @returns {boolean}
+     */
+    isNumpadDecimalSeparatorKey(ev) {
+        return ev.getModifierState('NumLock')
```

Comment:
I'm really not convinced about the modifierState, at least used alone.
I admit we may or may not support the following, but having numlock disabled and pushing shift + numpad dot works (it displays a dot).